lib/portfolio.js

```diff
@@ -181,6 +181,57 @@ module.exports = class Portfolio {
       }
     }
     return backtest;
+  } */
+
+  calculateHistoricalPerformance(candlestickData, curA, curB, metric) {
```

Comment:
I changed currencyPair to curA and curB. (Yuh) That way, we're not hardcoding the currency pairs.